ERCs/eip-5732/contracts/CommitToMintImpl.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: Apache-2.0
+// Author: Zainan Victor Zhou <zzn-ercref@zzn.im>
+// Visit our open source repo: http://zzn.li/ercref
+
+pragma solidity ^0.8.9;
+
+import "hardhat/console.sol";
+import "./IERC5732.sol";
+import "@openzeppelin/contracts/utils/introspection/ERC165.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+
+/**
```

Comment:
Should stick with either `/// @dev` style or `/*`, since I notice there is some inconsistency with this and `IREC5732.sol` comment style. The former style has grown more popular in the ecosystem so I would recommend that.

To see standard comment with all fields such as Title, Author, Dev, etc, here is an [example](https://github.com/mattstam/solidity-template/blob/1844397f96f7a0e9e7057bbf8b430b5fa08a93b5/contracts/interfaces/ICounter.sol#L4) that may be useful.

Reply:
Sounds good